src/parsers/STATICPOOLparser.py

```diff
@@ -0,0 +1,149 @@
+""" 
+parsers/STATICPOOLparser.py 
+
+A parser for a static pool. 
+
+Needs functionality: 
+
+- Given set of observations, update pool and training sets
+    - update_pool_and_train()
+- Gets train and validation datasets 
+    - get_train_val_dataset() 
+
+"""
+
+from .base import Parser
+import torch
+import pandas as pd
+from common import data_split, CSVLoader, HDFLoader, PickleLoader, apply_scaler
+
+
+class STATICPOOLparser(Parser):
+    """This is a dummy class as for the moment the sampler handles everything"""
+
+    def __init__(self, data_path: str, *args, **kwargs):
+        self.data_args = kwargs.get("data_args")
+        self.target_keys = kwargs.get("target")
+        self.input_keys = kwargs.get("inputs")
+
+        self.keep_keys = self.target_keys + self.input_keys
+        self.data = self.gather_data_from_storage(data_path)
+        self.data = self.data[self.keep_keys]
+        self.mapping_column_indices = self.get_column_idx_mapping()
+
+        self.input_col_idxs = [
+            self.mapping_column_indices[col_idx] for col_idx in self.input_keys
+        ]
+        self.output_col_idxs = [
+            self.mapping_column_indices[col_idx] for col_idx in self.target_keys
+        ]
+
+        self.train, self.valid, self.test, self.pool = data_split(
+            self.data, **self.data_args
+        )
+        self.train = self.train.sample(0)
+        self.train = torch.tensor(self.train.values)
+        self.valid = torch.tensor(self.valid.values)
+        self.test = torch.tensor(self.test.values)
+        self.pool = torch.tensor(self.pool.values)
+        # self.train, self.valid, self.test, self.pool, self.scaler = apply_scaler(self.train, self.valid, self.test, self.pool, scaler=None, op='transform')
```

Comment:
probably useful to delete this, as the current logic in the Executor requires the data in the Parser attributes to be unscaled